services/frontend-v3/src/i18n/en_CA.json

```diff
@@ -136,6 +136,7 @@
   "profile.acting.period.start.date": "Acting start date",
   "profile.address": "Address",
   "profile.branch": "Branch",
+  "profile.basic": "Basic Info",
```

Comment:
Where is this used? If its for the main card I don't think we need a title for the card.

Reply:
Would recommend Personal Information or Employee Information

Reply:
It's used for the anchor thing I made